app/services/AtsListService.scala

```diff
@@ -38,20 +38,20 @@ class AtsListService @Inject()(
   appConfig: ApplicationConfig)(implicit ec: ExecutionContext)
     extends AccountUtils {
 
-  def createModel(converter: (AtsListData => GenericViewModel))(
-    implicit hc: HeaderCarrier,
-    request: AuthenticatedRequest[_]): Future[GenericViewModel] =
-    getAtsYearList map {
-      checkCreateModel(_, converter)
-    }
-
-  private def checkCreateModel(
-    output: Either[Int, AtsListData],
-    converter: AtsListData => GenericViewModel): GenericViewModel =
-    output match {
-      case Right(atsList)  => converter(atsList)
-      case Left(NOT_FOUND) => new NoATSViewModel
-      case Left(_)         => new ATSUnavailableViewModel
+  def createModel()(implicit hc: HeaderCarrier, request: AuthenticatedRequest[_]): Future[Either[Int, AtsList]] =
+    getAtsYearList map { response =>
+      response match {
+        case Right(atsList) =>
+          Right(
+            AtsList(
+              atsList.utr,
+              atsList.taxPayer.get.taxpayer_name.get("forename"),
+              atsList.taxPayer.get.taxpayer_name.get("surname"),
```

Comment:
Just wondering why the names and UTR are extracted here?

Reply:
We are displaying the name and utr on the merge page